src/Handlebars/Context.php

```diff
@@ -216,6 +273,76 @@ public function get($variableName, $strict = false)
         return $current;
     }
 
+    /**
+     * Given a data variable, retrieves the value associated.
+     *
+     * @param $variableName
+     * @param bool $strict
+     * @return mixed
```

Comment:
Should include an @throws annotation indicating that a LogicException can be thrown.